app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+  before_action :require_admin
+
+  def require_admin
+    return if current_user&.role == "admin"
```

Comment:
```suggestion
    return if current_user&.role == "admin"
```
Chú tìm hiểu lại enum đi nhé
`current_user.admin?` ri thôi